src/main/java/de/bonndan/nivio/util/URIHelper.java

```diff
@@ -0,0 +1,25 @@
+package de.bonndan.nivio.util;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Optional;
+
+public class URIHelper {
+
+    /**
+     * Returns an URI object if the string is an URI and has host and scheme.
```

Comment:
Actually it returns an Optional of an URI object ;)

Reply:
done